lib/statement_identifier_service.rb

```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+
```

Comment:
Would it make sense to move this service to under the `lib/debt_management_center` parent folder? I realize it's not strictly a DMC associated service but would might help clarify code ownership with our debt team rather than a general purpose service under the root of `/lib`.